test/evergreen.yml

```diff
@@ -38,29 +38,17 @@ functions:
       remote_file: wiredtiger/${build_variant}/${revision}/artifacts/${dependent_task|compile}_${build_id}.tgz
       bucket: build_external
       extract_to: wiredtiger
-  "fetch artifacts from little-endian" :
+  "fetch artifacts from endian" :
     - command: s3.get
       params:
         aws_key: ${aws_key}
         aws_secret: ${aws_secret}
-        remote_file: wiredtiger/little-endian/${revision}/artifacts/WT_TEST.tgz
+        remote_file: wiredtiger/${endian_format}/${revision}/artifacts/WT_TEST.tgz
         bucket: build_external
-        local_file: WT_TEST-little-endian.tgz
+        local_file: WT_TEST-${endian_format}.tgz
     - command: archive.targz_extract
```

Comment:
I was wondering if we could use `extract_to` here? Like [this](https://github.com/wiredtiger/wiredtiger/pull/7278/files#diff-42f20e21217bd7d4b14489056e1ea24ea2b87599c85413d49182a3b0b721c132L40)

Reply:
My recollection is the `extract_to` setting has a built-in timer that could expire as the artifacts extraction could take a long time (esp on a big-endianness/zSeries machine). Feel free to give it a try with `extract_to` though.

Reply:
I believe @etienneptl  ran a patch build.

Reply:
I am still trying to use the `extract_to` function but no luck so far. I don't know if the Evergreen command cannot archive multiple folders at once or if I am not using it correctly. Will keep you updated.
As a side note, all patch builds are posted in the Jira ticket.

Reply:
I have added the `extract_to` command in 7b380b5234b79a9ad776492d7755dbac26b072bf.